enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,359 @@
+---
+title: set_eip_nlb_ingress
+authors:
+  - miheer
+reviewers:
+  - Miciah
+  - frobware
+  - gspence
+  - candace
+approvers:
+  - Miciah
+  - frobware
+  - gspence
+api-approvers:
+  - joel
+  - deads
+creation-date: 2024-03-06
+last-updated: 2024-03-22
+tracking-link:
+  - "https://issues.redhat.com/browse/NE-1274"
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Set AWS EIP For NLB Ingress Controller
+
+## Summary
+
+This enhancement allow user to set AWS EIP for the NLB default or custom ingress controller.
+This is a feature request to enhance the IngressController API to be able to support static IPs during
+- Install time
+- Custom NLB ingress controller creation 
+- Reconfiguration of the router.
+
+## Motivation
+
+### User Stories
+
+- As an administrator, I want to provision EIPs and use them with an NLB IngressController.
+- As a user or admin, I want to ensure EIPs are used with NLB on default router at install time.
+- As a user, I want to reconfigure default router to use EIPs.
+- As a user of OpenShift on AWS (ROSA), I want to use static IPs (and therefore AWS Elastic IPs) so that 
```

Comment:
These roles are ambiguous.  Is the administrator or user in all cases the cluster administrator?  If there are multiple roles, can you define them?

Reply:
OK. From my understanding we shall have only one user i.e the cluster-admin as he is the one who is responsible for creating routers.